src/api/faq/faq.controller.js

```diff
@@ -31,15 +31,10 @@ async function getFAQ(req, res) {
       }));
     return res.json(finalResponse);
   } catch (error) {
-    const {
-      message,
-    } = error;
-    const status = error instanceof ReferenceError ? 404 : 500;
-
-    return res.status(status).json({
-      status,
-      message,
-    });
+    if (error instanceof ReferenceError) {
+      res.status(404);
+    }
+    next(error);
```

Comment:
Causes linter error "Expected to return a value at the end of async function 'getFAQ'  consistent-return"

Should be
`return next(error);`